phivk/gradu.al
# Update/adds form components

Note: This is based of #126, so if there are changes needed to that PR, then this one will need to be rebased.

To test:
- Run `yarn install`
- Navigate to `/form`
- Complete the form 
- Submit the form
  - You should be redirected to the home page
  - You should be able to see the data in the Supabase instance
  - You should be able to see the new data in the Vuex store

Left to do:
- Add the search/combo box to the multiple select
- ~Improve the progress bar~
